### PR TITLE
Override pax logging version to address #1349.

### PR DESCRIPTION
**Pull request by splatch**
Signed-off-by: Łukasz Dywicki <luke@code-house.org>

### CODE REVIEW

**Comment by splatch** [comment]
Note - pax 2.0.10 is still listed in contents of archive, can be trimmed by ant or assembly mask. Runtime picks up 2.0.11 which is CVE clear. Feel free to backport it to 3.0.x and 3.1 versions. ;-)

File: distributions/openhab/processing.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<featuresProcessing xmlns="http://karaf.apache.org/xmlns/features-processing/v1.0.0">
+
+    <bundleReplacements>
+        <bundle originalUri="mvn:org.ops4j.pax.logging/pax-logging-api/[0,2.0.11)" replacement="mvn:org.ops4j.pax.logging/pax-logging-api/2.0.11" mode="maven" />
```
Comment (J-N-K) [inline distributions/openhab/processing.xml:5]: Why not use 2.0.12?

Comment (splatch) [reply distributions/openhab/processing.xml:5]: I could swear 2.0.11 was fine two days ago ;-), lets go with 2.0.12 then!

**Comment by kaikreuzer** [comment]
Thanks for this fix, @splatch!
I'll keep this as a fallback, if we run into any issues with https://github.com/openhab/openhab-distro/pull/1344.
But as Karaf 4.3.4 fixes some other issues as well, the plan would still be to release openHAB 3.2 directly with Karaf 4.3.4, so this additional fix won't be necessary.
Wrt backporting: Afaics, the criticality for openHAB isn't that high as log4j 2.15 already fixed the main/critical issue.
Assuming that most people on 3.x will anyhow update to 3.2, the effort to do yet another round of patch releases might not be worth it.

**Comment by jcz1** [comment]
FYI 2.0.11 is vulnerable to CVE-2021-45046 (score changed to 9.0 / critical).

https://github.com/ops4j/org.ops4j.pax.logging/issues/420

**Comment by wborn** [comment]
Thanks for the PR @splatch! I've made a few improvements in https://github.com/splatch/openhab-distro/pull/6, just in case the Karaf committee tries to spoil our release party. :partying_face:

**Comment by splatch** [comment]
Looks good, to be fair I didn't know the `exclude` stuff you used.
I don't think there are any show stoppers for Karaf 4.3.4 to get released, unless another surprise gets out of its dependencies ;-)

**Comment by splatch** [comment]
Solved through #1354

**Pull request closed without merging**